### DISCUSSION TITLE
What should I look after when purchasing a webcam whose image will go through a YOLOv8 model?

### DISCUSSION THREAD

**Discussion by fluffytid** [ultralytics/ultralytics#7148] [General]
I believe YOLOv8 will convert any image its given into 640x640, or is it 640x480?, that means getting a better camera with 1920x1080 or something like is a waste?

There are thousands of cameras in the markey and I don't really know what I should use to discriminate one from another

**Comment by RizwanMunawar** [comment]
Thanks for sharing the information @fluffytid! Yes, you are right, the default image size for Ultralytics YOLOv8 models is set to 640 x 640. However, it's important to note that this doesn't restrict you from utilizing the model for larger image sizes. If you have a camera with a resolution of 1280 * 720, you can set the `imgsz` argument value to 1280, enabling the model to process images of size 1280 x 720.


Hope this helps! Let me know if you have any additional questions.

Thanks and Regards
Ultralytics Team!

**Reply by fluffytid** [reply]
I presume with 1280*1280 it will take around 4 times more to proccess, right?

As a matter of fact I have a problem that the time is taking a bit longer than I wished for. Can you also set lower image sizes?

**Reply by RizwanMunawar** [reply]
Yes, right! An increase in imgsz will increase processing time. Yes, you can also decrease the image size i.e `imgsz=320`

Hope this helps! Let me know if you have any additional questions.

Thanks and Regards
Ultralytics Team!
